taxi_service/settings.py

```diff
@@ -140,3 +142,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap4"
```

Comment:
The `CRISPY_ALLOWED_TEMPLATE_PACKS` setting is correctly set to `bootstrap4`. Ensure that this is consistently used across all forms in the project.